docker_secrets.json

```diff
@@ -5,5 +5,9 @@
   "DB_USER": "",
   "DB_HOST": "",
   "DB_PORT": "",
-  "DB_PASSWORD": ""
+  "DB_PASSWORD": "",
+  "ALLOWED_HOSTS": [],
+  "SITE_ID": "",
+  "ADMINS": "",
+  "STATIC_ROOT": ""
 }
```

Comment:
Do we need to include `STATIC_URL` also?

Reply:
Oh, yes! I thought `STATIC_URL` in `coda/config/settings/base.py` should be fine, but looking at it again, as we are using `get_secret` to get the value in `production.py`, we'll require it in `secrets.json`. I'll have it added.